openmetadata-ui/src/main/resources/ui/src/components/ActivityFeed/ActivityFeedEditor/SendButton.tsx

```diff
@@ -39,12 +41,18 @@ export const SendButton: FC<SendButtonProp> = ({
       <Popover
         content={
           <Fragment>
-            <strong>Send now</strong>
+            <strong>{i18next.t('label.send-now')}</strong>
             <p>
-              Press{' '}
-              <kbd className="tw-bg-white tw-text-grey-body tw-rounded-sm tw-px-1 tw-py-0.5">
-                Return
-              </kbd>
+              {i18next.t('label.press')}
+              <Transi18next
+                i18nKey="message.tour-step-discover-all-assets-at-one-place"
+                renderElement={
+                  <kbd className="tw-bg-white tw-text-grey-body tw-rounded-sm tw-px-1 tw-py-0.5" />
+                }
+                values={{
+                  text: i18next.t('label.return'),
+                }}
+              />
             </p>
```

Comment:
as this this component, can we use `useTranslation` hook instead of` i18next.t`?